doc/dev/documenting.rst

```diff
@@ -2,6 +2,20 @@
  Documenting Ceph
 ==================
 
+User documentation
+==================
+
+The documentation on docs.ceph.com is generated from the restructuredText
+sources in ``/doc/`` in the Ceph git repository.
+
+Please make sure that your changes are written in a way that is intended
+for end users of the software, unless you are making additions in
+``/doc/dev/``, which is the section for developers.
+
+All pull requests that modify user-facing functionality must
+include corresponding updates to documentation: see 
+`Submitting Patches`_ for more detail.
+
```

Comment:
Since this is meant for general consumption, why not move it under start/documenting-ceph ?